### ISSUE TITLE
It's possible to create items with broken stats that overflow into stuff and break the game

### ISSUE THREAD

**Issue by qndel** [diasurgical/devilutionX#2127]
[single_30.zip](https://github.com/diasurgical/devilutionX/files/6603128/single_30.zip)
Here's a save where I'm shooting holy bolts from a bow (they consume mana in devilutionx)
My items are pretty much broken, we shouldn't allow such overflows

Here's an even harder one
![image](https://user-images.githubusercontent.com/14297035/120903528-dfd1a180-c646-11eb-88da-cf0471881b33.png)
![image](https://user-images.githubusercontent.com/14297035/120903531-e829dc80-c646-11eb-8e6c-89b5966c924f.png)

[single_30.zip](https://github.com/diasurgical/devilutionX/files/6603141/single_30.zip)

**Comment by AJenbo** [comment]
And this all works normally when loaded in vanilla?

**Comment by Chance4us** [comment]
> And this all works normally when loaded in vanilla?

Yes.
![1](https://user-images.githubusercontent.com/45902449/120903876-288a5a00-c649-11eb-8d02-cad9bfb2d6e7.png)
![2](https://user-images.githubusercontent.com/45902449/120903881-2d4f0e00-c649-11eb-837c-14f6332930f8.png)

**Comment by AJenbo** [comment]
Looks like the same issues...

**Comment by Chance4us** [comment]
Only I could not shoot holy bolts in vanilla, but in devilutionx.

**Comment by AJenbo** [comment]
Yeah I think I fixed that arrow type not working correctly, but normally you can't get any weapon with it, so :shrug:

**Comment by qndel** [comment]
I'm shooting holy bolts in vanilla just fine

**Comment by AJenbo** [comment]
![image](https://github.com/diasurgical/devilutionX/assets/204594/aab0dcf2-5e00-497b-99de-fcfde592348e)
I think this is a good sign?

**Comment by AJenbo** [comment]
@qndel could you fill us in on which vallues the overflow is being employed for?
